INSTALL.md

```diff
@@ -79,39 +84,16 @@ You will need to update the following configuration files:
         # ...
     ```
 
-### Image Registry Configuration
```

Comment:
I agree with removing this in the streamlined initial install instructions. It is pretty verbose.

That said, I think there is a lot of nuance here that we should keep around in some form -- maybe in a separate `registry-configuration.md` doc. 🤔 

Like `DockerHub` has some pretty significant deviations from other OCI registries, but the others have minor differences as well between each other (ECR requires different auth for example)  and it would be good to have a place to document all of that.

Reply:
We already have `using-container-registry-signed-by-custom-ca.md` and that could probably consolidated into a more general registry config doc as well.

Reply:
If we feel like we need to document the different nuances, it makes sense to have a separate document. Alternatively we could have a section at the bottom? Something like _Notes on container registries_?